script/foreman-debug

```diff
@@ -402,4 +411,5 @@ fi
 
 printv "Finished in $SECONDS seconds"
 
+clean_temp
```

Comment:
Why calling it here? Don't we want to clean the things up only, when we create the temporary dirs by ourselves? If the user provides their own dir, we should probably keep it for them.

Given we register the fucntion using `trap`, it would actually get called twice.

Reply:
I don't know why, but trap does not call on script exit. I could not figure it out how to do it, if you can please teach me.

Reply:
The script keeps the dir if NOTAR is submitted, otherwise it deletes it if it's in /var/tmp. So it actually works the way you like for the most part.

Reply:
I have not seen this behavior: was testing this on rhel7 and the trap was working just fine. With this around, it was printing the 'cleaning' twice in the output, which is quite odd. Could you check again we really need this? I would be curious about the reproducer.

Reply:
Could you re-test need for this: I have not seen an evidence that this actually needs to be called.